### DISCUSSION TITLE
Problem 3b - Task05

### DISCUSSION THREAD

**Discussion by SMRazaRizvi96** [chulminy/CIVE497-CIVE700#164] [Q&A]
For Problem 3b - Task05, instead of using the `fspecial` function in MATLAB, can we use the function created in Problem 3a, generate h1 and h2, and apply them to an image to compare the effects of h1 and h2?

**Comment by chulminy** [comment]
Yes. You can.

**Reply by chulminy** [reply]
Yes. You just answer the question based on your implementation. 

Chul Min
